### ISSUE TITLE
is there any live instance for users to poke at? :) 

### ISSUE THREAD

**Issue by cartazio** [liamoc/dixi#16]

**Comment by liamoc** [comment]
No, sorry. It will be possible once I have administrator controls added, but without that feature it's too easy for people to muck it up and it's impossible for me to remove content once it's added (i.e. even if i revert patches they're still in the history)
